IntelliTect.TestTools.TestFramework/IntelliTect.TestTools.TestFramework.Tests/TestBase.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Xunit;
+
+namespace IntelliTect.TestTools.TestFramework.Tests
+{
+    public class TestBase
```

Comment:
Should this class be static since it only contains a static method, or are other classes expected to derive from it?

Reply:
Second option; a few other classes derive from this.